src/query/expression/src/types/boolean.rs

```diff
@@ -36,10 +37,10 @@ pub struct BooleanType;
 impl ValueType for BooleanType {
     type Scalar = bool;
     type ScalarRef<'a> = bool;
-    type Column = Bitmap;
+    type Column = BooleanBuffer;
```

Comment:
Why not use `NullBuffer`, it has materialized null_count.

Reply:
I can not find a builder for `NullBuffer` and it does not `impl BitAnd`

Reply:
1. `NullBuffer::new()`
2. `pub fn union(lhs: Option<&NullBuffer>, rhs: Option<&NullBuffer>) -> Option<NullBuffer> `

Reply:
Also: 
`impl From<BooleanBuffer> for NullBuffer`

Reply:
But these all operation is direct on `BooleanBuffer`: it implments BitAnd, it provides `value_unchecked`, and have `BooleanBufferBuilder`. I think `BoolteanBuffer` is the directly replacement for `Bitmap` in arrow2.